versions/1.0.0.md

```diff
@@ -70,6 +70,17 @@ The list of actions MUST be applied in sequential order to ensure a consistent o
 
 The `extends` property can be used to indicate that the Overlay was designed to update a specific [[OpenAPI]] document. Where no `extends` is provided it is the responsibility of tooling to apply the Overlay document to the appropriate OpenAPI document(s).
 
+In the following example the `extends` property specifies that the overlay is designed to update the Redocly Museum OpenAPI document.
+
+```yaml
+overlay: 1.0.0
+info:
+  title: Overlay for the Redocly Museum API
+  version: 1.0.0
+extends: 'https://raw.githubusercontent.com/Redocly/museum-openapi-example/refs/heads/main/openapi.yaml'
```

Comment:
Could we choose something vendor-neutral, for example one of these: https://github.com/OAI/learn.openapis.org/tree/main/examples